prow/plugins/config.go

```diff
@@ -420,6 +420,15 @@ type ConfigMapSpec struct {
 	GZIP *bool `json:"gzip,omitempty"`
 	// Namespaces is the fully resolved list of Namespaces to deploy the ConfigMap in
 	Namespaces []string `json:"-"`
+	// Cluster in which the configMap needs to be deployed. If no cluster is specified
+	// it will be deployed to "default".
+	Cluster string `json:"cluster,omitempty"`
+	// Clusters in which the configMap needs to be deployed, in addition to the above
+	// cluster provided, or the default if it is not set.
+	AdditionalClusters []string `json:"additional_clusters,omitempty"`
+	// Clusters is the fully resolved list of Clusters to deploy the ConfigMap in
+	// Together with Namespaces, the configmap is updated for each ns in Namespaces and for each c in Clusters
+	Clusters []string `json:"-"`
```

Comment:
I'm wondering: do we *need* to follow the `cluster/additional_clusters` pattern? I assume for namespaces there was `namespace:` first, then `additional_namespaces` was added to keep configs backwards compatible, but why can't we simply have `clusters` here as a single list, defaulting to the defautl cluster if that list is empty?

Reply:
Done.